vowpalwabbit/conditional_contextual_bandit.cc

```diff
@@ -130,7 +130,7 @@ bool split_multi_example_and_stash_labels(const multi_ex& examples, ccb& data)
       case example_type::slot:
         data.slots.push_back(ex);
         break;
-      default:
+      case unset:
```

Comment:
what is unset

Reply:
It is the other enum value

Reply:
we dont need a default case anymore?

Reply:
Yeah now that I look at this again it seems perhaps the `default` was better and less likely to be misused in future